### DISCUSSION TITLE
Task Description Rendering

### DISCUSSION THREAD

**Discussion by ArthurThiam** [jamiebrynes7/obsidian-todoist-plugin#141] [Ideas]
I'm loving this plugin! Currently the only things that can be pulled over from todoist (in addition to the task itself), are the date, project/section, and labels. I personally don't extensively use these, but I was wondering if it would be difficult to also render task descriptions? I'm not sure if these were added to todoist after the last release but it would be a great addition to the plugin!

**Comment by jamiebrynes7** [comment]
Hey @ArthurThiam, this is tracked in https://github.com/jamiebrynes7/obsidian-todoist-plugin/issues/147 :)
